# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1306343508 25200
#      Wed May 25 10:11:48 2011 -0700
# Branch stable
# Node ID 02f8f4c509ec082720784f5970e19d1d080ba659
# Parent  486c9aea34d322a980b91df40fbac293254d4b92
ls_command.m: Add additional calling forms to DOCSTRING.

* ls_command.m: Add additional calling forms to DOCSTRING.

diff --git a/scripts/miscellaneous/ls_command.m b/scripts/miscellaneous/ls_command.m
--- a/scripts/miscellaneous/ls_command.m
+++ b/scripts/miscellaneous/ls_command.m
@@ -12,20 +12,19 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {[@var{old_cmd} =} ls_command (@var{cmd})
-## Set or return the shell command used by Octave's @code{ls} command.
-## The value of @var{cmd} must be a character string.
-## With no arguments, simply return the previous value.
+## @deftypefn  {Function File} {@var{val} =} ls_command ()
+## @deftypefnx {Function File} {@var{old_val} =} ls_command (@var{new_val})
+## Query or set the shell command used by Octave's @code{ls} command.
 ## @seealso{ls}
 ## @end deftypefn
 
 ## Author: jwe
 
 function old_cmd = ls_command (cmd)
 
   global __ls_command__;
